src/check-c-linkage-decls.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import sys, os
+
+os.chdir (os.environ.get ('srcdir', os.path.dirname (__file__)))
+
+HBHEADERS = os.environ.get ('HBHEADERS', '')
+if len (HBHEADERS):
+	HBHEADERS = [x for x in HBHEADERS.split (' ') if x]
```

Comment:
Just use `HBHEADERS.split ()`.  In other places as well.

Reply:
needed to trim empty entries passed by autotools

Reply:
Did you try what I suggested?

Reply:
`.split()` by default does exactly what you are doing here.

Reply:
wow, nice one, done